packages/gestalt/src/Link/InternalLink.tsx

```diff
@@ -52,7 +52,7 @@ type Props = {
   tabIndex: -1 | 0;
   rounding?: Rounding;
   selected?: boolean;
-  size?: 'sm' | 'md' | 'lg';
+  size?: 'sm' | 'md' | 'lg' | 'social';
```

Comment:
can you move social to  wrappedComponent: 'buttonsocial'?

I don't want this component to be intertwined with the whole Button configuration